nailgun/entities.py

```diff
@@ -844,9 +844,11 @@ def __init__(self, server_config=None, **kwargs):
             'command': entity_fields.StringField(
                 required=True,
                 str_type='latin1',
+                default='top',
```

Comment:
The only concern is that top is not common and there are some images that does have it. Which will cause an error.

We have a place in robottelo to add some default values [1]. To me there seems to be a better place to define these default values.

If @oshtaier and @Ichimonji10 thinks this is a good addition, I will not block it. I'm only not completely sure about this being on NailGun.

[1] https://github.com/SatelliteQE/robottelo/blob/master/robottelo/common/helpers.py#L375